services/account-without-juggler/repositories/account/datasources/mysqlconn.ts

```diff
@@ -0,0 +1,201 @@
+const debug = require("debug")("account-without-juggler");
```

Comment:
const debug = require('debug')('loopback:repositories:account:datasources:connections:mysql');

Single quote everything in this file too